cloudhub-2/modules/ROOT/pages/ch2-architecture.adoc

```diff
@@ -159,7 +159,9 @@ CloudHub 2.0 backend services determine the values of:
 _uniq-id_::
```

Comment:
@sarathecoubian what would be the best place to add a proper example? IMO we should at some point offer a real example and not just the shard ref in the URL.

Reply:
We can put examples in these definitions as long as we can provide examples for both uniq-id and shard. Location matters less than consistency.

Reply:
@sarathecoubian can the docs team work on adding that example and updating shard docs for Shared space apps everywhere else too. Here is the related GUS ticket i created for the action item: https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001VkuYaYAJ/view. Thanks 🙏